packages/js/src/ui/components/Header/StatusOptions.tsx

```diff
@@ -16,33 +15,24 @@ export const StatusOptions = (props: { setFeedOptions: (options: FetchFeedArgs)
     <>
       <StatusItem
         label={DropdownStatus.UnreadRead}
-        /**
-         * TODO: Implement setFeedOptions and isSelected after Filter is implemented
-         */
         onClick={() => {
-          props.setFeedOptions({ status: NotificationStatus.UNREAD });
+          props.setFeedOptions({ archived: false });
```

Comment:
✏ **suggestion:** No props will fetch all notifications and we can use that for the label.

Reply:
For that label option we need to fetch unread and read only, so i guess we don't need to fetch the archived messages

Reply:
ahh yes, you are right! 🙌